README.md

```diff
@@ -1,583 +1,317 @@
 
-```shell
-# Create a kind cluster
-kind create cluster
+- `--preserve-import-path` (`-P`) will include the entire importpath: `my.registry/repo/github.com/my-user/my-repo/cmd/app`
+- `--base` (`-B`) will omit the MD5 portion: `my.registry/repo/app`
+- `--bare` will only include the `KO_DOCKER_REPO`: `my.registry/repo`
 
-# Deploy to kind w/o registry.
-KO_DOCKER_REPO=kind.local ko apply -f config/
-```
+## Local Publishing Options
 
-If you want to create a `kind` cluster with a non default name, you can set the
-`KIND_CLUSTER_NAME` variable to the respective name (which is also supported by
-[`kind` itself](https://github.com/kubernetes-sigs/kind/releases/tag/v0.8.0)).
+`ko` is normally used to publish images to container image registries, identified by `KO_DOCKER_REPO`.
 
-Like with `minikube` above, a caveat of this approach is that it will not work
-if your container is configured with `imagePullPolicy: Always` because despite
-having the image locally, a pull is performed to ensure we have the latest
-version, it still exists, and that access hasn't been revoked. A workaround for
-this is to use `imagePullPolicy: IfNotPresent`, which should work well with `ko`
-in all contexts.
+`ko` can also publish images to a local Docker daemon, if available, by setting `KO_DOCKER_REPO=ko.local`.
 
-Note that images will not appear in the Docker daemon running `kind` as the
-cluster itself is running in a container that is running `containerd` inside.
-The images are loaded into the respective `containerd` daemon.
+`ko` can also publish images to a local [KinD](https://kind.sigs.k8s.io) cluster, if available, by setting `KO_DOCKER_REPO=kind.local`.
 
-## Configuration via `.ko.yaml`
+## Multi-Platform Images
 
-While `ko` aims to have zero configuration, there are certain scenarios where
-you will want to override `ko`'s default behavior. This is done via `.ko.yaml`.
+Because Go supports cross-compilation to other CPU architectures and operating systems, `ko` excels at producing multi-platform images. This allows you to build an image that can easily be run on other architectures such as arm64, s390x, PowerPC, etc., or other OSes like Windows.
 
-`.ko.yaml` is put into the directory from which `ko` will be invoked. One can
-override the directory with the `KO_CONFIG_PATH` environment variable.
+To build and push an image for all platforms supported by the configured base image, simply add `--platform=all`. This will instruct `ko` to look up all the supported platforms in the base image, execute `GOOS=<os> GOARCH=<arch> GOARM=<arm> go build` for each platform, and produce a manifest list containing an image for each platform.
 
-If neither is present, then `ko` will rely on its default behaviors.
+You can also select specific platforms, for example, `--platform=linux/amd64,linux/arm64`
 
-### Overriding the default base image
+## Static Assets
```

Comment:
Some feedback from @ahmetb was that it's not completely obvious how to include symlinks in here, which is really useful for stuff like including .git refs or not having to restructure your entire repo to conform to `ko`.

Reply:
Good call, added a tip.